### ISSUE TITLE
Projects to include for data publication

### ISSUE THREAD

**Issue by peterdesmet** [inbo/etn-occurrences#7]
Overview of the projects that will be included in the dataset we will publish:

animal project | scientific name | indiv | detections | include in dataset
--- | --- | --- | --- | ---
2011 Rivierprik | Lampetra fluviatilis | 39 | 114.605 | yes
2012 Leopoldkanaal | Anguilla anguilla | 104 | 2.215.830 | yes
2013 Albertkanaal | Anguilla anguilla | 161 | 4.784.914 | yes
2013 Albertkanaal | Salmo salar | 135 | 1.375.687 | yes
2013 Albertkanaal | Sync tag | 13 | 16M | no, not biological
2014 Demer | Petromyzon marinus | 2 | 42 | yes
2014 Demer | Rutilus rutilus | 2 | 11.030 | yes
2014 Demer | Silurus glanis | 9 | 86.973 | yes
2014 Demer | Squalius cephalus | 3 | 139.013 | yes
2015 Dijle | Anguilla anguilla | 1 | 41.798 | yes
2015 Dijle | Cyprinus carpio | 2 | 4.944 | yes
2015 Dijle | Platichthys flesus | 8 | 101.488 | yes
2015 Dijle | Rutilus rutilus | 4 | 7.870 | yes
2015 Dijle | Silurus glanis | 11 | 78.501 | yes
2015 Fint | Alosa fallax | 44 | 241.530 | no, not yet
2015 PhD Verhelst | Anguilla anguilla | 136 | 1.696.255 | yes
2015 PhD Verhelst | Gadus morhua | 106 | 1.526.911 | no, not yet
Homarus | Homarus gammarus | 2 | 22.265 | yes
PhD Jan Reubens | Gadus morhua | 41 | 709.912 | yes
PhD Jan Reubens | Sentinel | 27 | 594.301 | no, not biological

**Comment by peterdesmet** [comment]
@PieterjanVerhelst @IPauwels @LienReyserhove this issue was originally posted at https://github.com/inbo/etn/issues/68 but then turned into a discussion about ghost detections. Can you check/discuss the above again? Does this include all data from PJ's PhD?

**Comment by PieterjanVerhelst** [comment]
@peterdesmet I found other values for the different columns. Also, in the data I requested via the ETN R-app, Petromyzon marinus was attributed to the 2015 Dijle project instead of 2014 Demer. However, in the ETN database, Petromyzon marinus is a species tagged in 2014 Demer.
@IPauwels, could you comment on this?

Further, the data from the Schelde eels from my PhD is not included. I am still planning of working with that data the coming year, but wouldn't mind to include it in the dataset-publication.

**Comment by peterdesmet** [comment]
The reason why you have different data is because my table was old. 😄 I've moved your table to the issue body. The only remaining difference is the 2012 Leopoldkanaal which was not in your list.

Would be nice to include your PhD data data! 👏 What are the project codes for those?

**Comment by PieterjanVerhelst** [comment]
I wrongly used the information from the network codes to provide an update of the table above. Therefore, I checked the info again, but this time for the animal codes. I also included the data from my PhD (animal code: 2015_phd_verhelst). @peterdesmet, is it possible to replace the table above by the table given here? Just note that the end date of the 2012_Leopoldkanaal project is wrong due to ghost detections. Also note that some eels tagged during my PhD are still being tracked so therefore the study is still ongoing.

**Comment by peterdesmet** [comment]
Table replaced at the top and removed from your comment to keep it easy to read (all of this is versioned).

**Comment by IPauwels** [comment]
Indeed, the sea lampreys were tagged in the Demer project. I did not check the table again regarding the number of individuals per species per project. Please let me know if it is better to double check it again.. 

So, if I am correct, we now have:
- PhD Verhelst (which entails most detections on Westerschelde and Zeeschelde, apart from Albertkanaal): 
paper - yes 
dataset - no (not yet, study still ongoing)
- Demer: 
paper - yes
dataset - yes
- Dijle: 
paper - yes
dataset - yes
- Albertkanaal/Maas (this is partly PhD Verhelst and partly PhD Vergeynst): for both: 
paper - yes
dataset - no (studies still ongoing)
- Leopoldkanaal: 
paper - yes
dataset - yes
- Bovenschelde (River lamprey tagging): 
paper - yes
dataset - yes
- PhD Reubens --> is this still correct?? I thought we would as well publish it and mention it in the datapaper?
paper - yes
dataset - yes 
- what about our shad data? --> definitely not publish them in the dataset, but will we mention them in the data paper, what do you think? I would say: yes?

This is how I thought we would publish everything, do you agree?

**Comment by peterdesmet** [comment]
Discussed in person with @IPauwels and @PieterjanVerhelst:

- Updates since https://github.com/inbo/etn-occurrences/issues/7#issuecomment-467088629: `2015 PhD Verhelst` + `2013 Albertkanaal` data will be included (assuming dataset will only be published in June).
- I created an overview of animal projects and in which network projects individuals were detected in a separate Google spreadsheet, but table above has been updated.

Remaining question for @jreubens:

- Can the `Homarus` animal project be included?
- Can the `PhD Jan Reubens` animal project be included (709.912 detections of Gadus morhua)
